systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -434,4 +411,49 @@ public static void copyImagePullSecret(String namespace) {
                 .withData(Collections.singletonMap(".dockerconfigjson", pullSecret.getData().get(".dockerconfigjson")))
                 .build());
     }
+
+    private static JsonArray getJsonArrayFromLog(String log) {
```

Comment:
JFYI: If you are making the parser the great way how to make it quickly only by defining excepted structure and 
 then just generate it by `ANTLR`...

Reply:
Good to know 🤔 thanks!